frontend/views/pages/Join.vue

```diff
@@ -19,8 +19,8 @@ div
         h1.is-title-1.c-title(data-test='groupName') {{ ephemeral.invitation.groupName }}
         p.has-text-1(data-test='invitationMessage') {{ ephemeral.invitation.message }}
       .card
-        signup-form(v-if='isStatus("SIGNING")' @submit-succeeded='accept')
-        login-form(v-else @submit-succeeded='accept')
+        signup-form(v-if='isStatus("SIGNING")' :postSubmit='accept')
+        login-form(v-else :postSubmit='accept')
```

Comment:
I don't understand how this change solves the problem.

The two lines `await this.postSubmit()` and `this.$emit('submit-succeeded')` are right next to each other, so what is the difference?

Reply:
Nice question. What I did here is to make the `Submit` button loading until the user finish to join the group not only SignUp or SignIn. Actually, the new solution is related to the way of `ButtonSubmit` component works.
The loading spinner will be cycling until the `click` listener of that `ButtonSubmit` component is finished.
If the SignUp form emit an event and `Join` stuff is processed on `Join` page by listening this event, the `ButtonSubmit` spinner will be cycling only until that event is emitted, not until the `Join` process is finished.

Reply:
Ah, I see, so it's because it waits on the async function `accept` before proceeding. Could you then add a comment here linking to that issue and explaining that we use `postSubmit` instead of `@submit-succeeded` because it supports waiting on async functions?

Reply:
Okay. Will do. I think I should make comments to the props of `SignUpForm` and `LoginForm` as well.